website/for_teachers.py

```diff
@@ -971,32 +974,26 @@ def parse_preview_adventure(self):
             return gettext("something_went_wrong_keyword_parsing"), 400
         return {"code": code}, 200
 
-    @route("/create_adventure", methods=["POST"])
+    @route("/create-adventure", methods=["GET"])
```

Comment:
Please keep this as a POST method since it indicates what this function actually does. You can still return whatever you want (the id in this case!)